active-record/README.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Active Record

```

Comment:
If you have a look at newer patterns, there is more info in the table. Here is Hugo's
```---
layout: pattern
title: Currying
folder: currying
permalink: /patterns/currying/
categories: Functional
language: en
tags: 
 - Decoupling
---
```